src/utils/tokens.js

```diff
@@ -0,0 +1,31 @@
+
+import sendJson from '../tmp_fetch_send_json'
+import { ACCOUNT_HELPER_URL } from './wallet'
+import * as Sentry from '@sentry/browser'
+import { wallet } from './wallet'
+
+export const getLikelyContracts = async (accountId) => (
+    await logAndIgnoreError(sendJson('GET', `${ACCOUNT_HELPER_URL}/account/${accountId}/likelyTokens`))
+)
+
+export const getMetadata = async (contract, accountId) => {
+    const account = await wallet.getAccountBasic(accountId)
+
+    let metadata = await logAndIgnoreError(account.viewFunction(contract, 'ft_metadata'))
```

Comment:
:nit: For consistency, if we don't need to re-assign the value of a variable, we should use `const`.  No big deal here since the entire variable is optional so its lifecycle is tiny, but we should enforce this with `eslint` so that it's auto-fixed for us on save if we forget and doesn't come up in code reviews as a nit/lint fix.

Reply:
Done